tests/resources/session-tests-template/modules-0/forms-0.xml

```diff
@@ -0,0 +1,41 @@
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
```

Comment:
We don't actually use the data in these forms, right?

Can we replace them all with just hte one template form?

Again, lean/less confusing.